chia/util/misc.py

```diff
@@ -472,3 +473,36 @@ def satisfies_hint(obj: T, type_hint: Type[T]) -> bool:
             if type(obj) is not type_hint:
                 return False
     return True
+
+
+async def load_plugin_configurations(root_path: str, config_type: str) -> List[str]:
```

Comment:
May as well take a `Path` I think.

```suggestion
async def load_plugin_configurations(root_path: Path, config_type: str) -> List[str]:
```

Reply:
fixed in next commit